scipy/spatial/transform/rotation.py

```diff
@@ -131,54 +132,58 @@ def as_dcm(self):
             return dcm
 
     @classmethod
-    def from_dcm(cls, mat, scalar=True):
-        # For now, mat must be a 3D matrix
-        num_rotations = mat.shape[0]
-        if scalar:
-            quat = np.empty((num_rotations, 4))
-            for rot_num in range(num_rotations):
-                A = mat[rot_num]
-                trA = A.trace()
-                choice = np.argmax(np.array([A[0, 0], A[1, 1], A[2, 2], trA]))
-
-                if choice in [0, 1, 2]:
-                    i = choice
-                    j = (i + 1) % 3
-                    k = (j + 1) % 3
-                    quat[rot_num, i] = 1 - trA + 2 * A[i, i]
-                    quat[rot_num, j] = A[j, i] + A[i, j]
-                    quat[rot_num, k] = A[k, i] + A[i, k]
-                    quat[rot_num, 3] = A[k, j] - A[j, k]
-                else:
-                    quat[rot_num] = np.array([
-                        A[2, 1] - A[1, 2],
-                        A[0, 2] - A[2, 0],
-                        A[1, 0] - A[0, 1],
-                        1 + trA])
+    def from_dcm(cls, dcm):
+        """Initialize rotation from direction cosine matrix.
+
+        This classmethod return a `Rotation` object from the input direction
+        cosine matrices. If the input matrix is not orthogonal, this method
+        creates the best approximation of the rotation.
+
+        Parameters
+        ----------
+        dcm : array_like, shape (N, 3, 3) or (3, 3)
+            Each `dcm[i]` corresponds to a `(3, 3)` direction cosine matrix.
```

Comment:
I would probably write: "A single matrix or a stack of matrices, where `dcm[i]` is  an i-th matrix." Or we can leave polishing docstrings for later.

Reply:
I'll make this change now itself.